### DISCUSSION TITLE
"May/June 1922: Friedmann Imagines a Changing Universe — to Einstein’s Chagrin"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#223] [General]
A bit of history, things we already know.
["May/June 1922: Friedmann Imagines a Changing Universe — to Einstein’s Chagrin"](https://www.aps.org/apsnews/2024/05/friedmann-universe-expand-contract?utm_source=may-amn&utm_medium=email&utm_campaign=may-amn)
Alexander Friedmann used Einstein’s general theory of relativity to propose a universe that could expand or contract.

**Comment by genotics** [comment]
This is an interesting side note to history. I thought that Einstein had introduced the cosmological constant in a paper in which he compared it to the Seeliger term, but perhaps that was an earlier study.
Meanwhile, a bit of history that might be of more direct interest to our discussions involves Nernst, and it appears that few are aware of it: <http://redshift.vif.com/JournalFiles/Pre2001/V02NO3PDF/V02N3INT.PDF>

**Reply by LifeIsStrange** [reply]
>  The creation of new matter is a necessity in a Universe which is in a state of
equilibrium.

An alternative to matter creation/destruction is matter/energy conversion/recycling. 
Quantum mechanics predicts and is experimentally confirmed, the phenomena of energy conversion to matter via electron pair production (which can create neutrons, protons and then via e.g. fusion, all other elements in stars or in fusors).
The theoretical temperature needed is already reached in the center of giants stars or neutron stars, moreover the often quoted temperature for neutron production is an upper bound, meaning there are countless catalytic mechanisms to produce them at lower temperature (pressure, magnetohydronymanics, magnetic reconnection, special topologies, convexion and many many niche effects I have seen on high energy physics papers that alter the efficiency by order of magnitudes).
As such the idea of potent conversion of energy to matter in the center of regular stars is in my neophyte understanding, a very credible possibility despite being a utterly ignored topic by the scientific research, which is extremely surprising as it is a continuation of the classical solar cycles.
The biggest unknown is: does it makes sense quantitatively? 
1) is the electron pair/neutron/proton production efficiency sufficent to makes the universe homeostatic
2) besides the process of conversion, how efficiently is light/cosmic rays reabsorbed in foreign stars
IMO it is point 2) that is the biggest difficulty for an homeostatic universe that doesn't suppose spontaneous matter creation.
Assuming a tired light mechanism, the light radiation could be converted to a particle e.g. a neutrino with a mass, that could be more efficiently gravitationally recaptured by stars but this is speculative until someone does a proper mathematical model that quantifies those qualitatives arguments.
It is also related to olber paradox and modified gravities/inertia and most importantly to the solar corona heat excess and to the faint young sun paradox.
btw as electron pair produce antiparticles, what would happen to them?

**Reply by Francois-Zinserling** [reply]
@LifeIsStrange 
My reply not really on point with the main discussion, but just to stimulate thought ...

> An alternative to matter creation/destruction is matter/energy conversion/recycling.

Electron/positron particle pair generation and annihilation is mostly undisputed and is fairly well observed. I shrug at the claim that protons and anti-protons are 'readily' produced though, while at the same time in QED and QCD the proposed sub-content of a proton would make it statistically nearly-impossible to produce 'by accident'. 

> btw as electron pair produce antiparticles, what would happen to them?

By the rules of particle production, conservation of charge, energy and momentum are foremost (in a classic setting)
**Charge Rule**: Explains why generally an electron and positron is observed, but does not forbid the creation of e.g. a zero charge neutron, or even a neutral atom.
**Energy Rule**: Electron and positron needs to be able to annihilate again to 0 mass, hence needed exact equal masses, although opposite charge. Here atom or neutron is forbidden because it doesn't decay to 0 mass, as proton and electron are typical decay products but (a) don't annihilate and (b) cannot annihilate to 0 mass. 

It is however allowed for an electron to have total energy $E = γmc^2$ content to equal the mass of a proton, so it is a possibility that 'neutrons' could form from a ZPF, decay immediately back to ZPF, **or** decay into electron and proton with equal energies, electrons lose kinetic energy rapidly due to collisions and protons remain stable. There-after cannot annihilate because electron energy is too low and energy rule would be violated.
By another small stretch of the imagination, and also elsewhere discussed, ZPF content can be ejected from a black hole since it has no mass. So we could (a) set quasars as the origins of the ZPF (=aether) and (b) look for matter generation in quasar jets.

Purely some speculation though, but would also like to know how much of this can be falsified.
I think I got a bit carried away though :)

**Comment by LifeIsStrange** [comment]
> A bit of history, things we already know. ["May/June 1922: Friedmann Imagines a Changing Universe — to Einstein’s Chagrin"](https://www.aps.org/apsnews/2024/05/friedmann-universe-expand-contract?utm_source=may-amn&utm_medium=email&utm_campaign=may-amn) Alexander Friedmann used Einstein’s general theory of relativity to propose a universe that could expand or contract.

> Einstein would later call the cosmological constant his “greatest blunder.”

This is a myth repeated ad nauseam, the greatest blunder is only testified by Gamov, that has already empirically been found to lie and greatly exaggerate about his relationship with Einstein.
Also since the cosmological constant allows for a static universe, and being a prime proponent of expansion, he is in conflict of interest.
Einstein did criticize that the cosmological constant was reprensented independently of gravity (not unified) but to call that his greatest blunder is a stretch for a simple formalism that allows both expanding and static universes to work (dark energy).

**Comment by genotics** [comment]
Good day Francois,
I find your speculations stimulating, and wonder if the following
exposition might resonate with your ideas:

[begin quote]
We now return to de Broglie's thermodynamics, which, as noted earlier,
implies an exchange of energy between particles and the vacuum via a
"vacuum thermostat". This idea can be developed as follows.

We saw earlier that the electron can be described by a model incorporating
a spherical surface for interactions with the surrounding space.

This surface seems to be the locus of interactions at the quark level. It
is also the site where particle charge quanta are stored; it is the core of
rest-mass particles. The surface determines the inner structure of a
counterpart of a Schwarzschild singularity in the quantum space-time of
particles. We will therefore treat this area as an absolute sink for vacuum
energy. This is in accordance with the analysis of Carroll (1991) who
concludes that "in addition to the motion of particle spin, there exists a
source-sink flow at right angles to the motion of spin-through the vortex
centre".

We assume that the vacuum space has an energy density $\rho_{0}$ (in terms
of mass).

It was established above, in the section dealing with the force action,
that the vacuum tube corresponds to a surface which has a radius equal to
the Plank radius. We can therefore also calculate the corresponding
"graviton" mass density in the vacuum from the relation between surface and
mass:

$$
A=\frac{\text { Surface }}{\text { Mass }}=\frac{\Phi_{P l}}{\Phi_{P l}
\cdot c T_{0} \cdot \rho_{0}}
$$

We thus have

$$
A \cdot c T_{0} \cdot \rho_{0}=1
$$

The density is:

$$
\rho_{0}=\frac{1}{A \cdot c T_{0}} \equiv \frac{2 \pi G}{A^{2} c^{2}}
$$

The energy inflow through the absorbing surface is:

$$
\frac{\mathrm{d} m_{e}}{\mathrm{~d} t}=c \rho_{0} \cdot A m_{e}
$$

The increase of energy causes the quantum-system's temperature to increase,
and at an equilibrium temperature it will radiate off energy with a Planck
spectrum.

As the radiating surface, we will use the surface corresponding to the time
constant in the electron system defined in the preceding section on the
fine structure constant. This time constant is assumed to be equal to the
length of the spiral in the electron system. The radiating surface is
consequently based on the quantum counterpart of the Compton wavelength:

$$
\Phi_{r a d}=\frac{1}{\pi} \ell^{2} \equiv
\frac{1}{\pi}\left(\frac{\hbar}{c m_{e}}\right)^{2}
$$

The radiation temperature depends on the relation between the absorbing and
the radiating surfaces:

$$
\frac{\Phi_{a b s}}{\Phi_{r a d}}=\frac{A m_{e}}{\frac{1}{\pi} \ell^{2}}
\equiv \frac{\alpha^{2}}{4}
$$

The temperature can be calculated with Stefan-Boltzmann's law:

$$
T=\sqrt[4]{\frac{\frac{\mathrm{d} m}{\mathrm{~d} t} \cdot c^{2}}{\sigma
\cdot \Phi_{\mathrm{rad}}}} \equiv \sqrt[4]{\frac{c^{3} \rho_{0}}{\sigma}
\cdot \frac{\Phi_{a b s}}{\Phi_{\mathrm{rad}}}}
$$

The radiation temperature is, finally:

$$
T=\sqrt[4]{\frac{2 \pi G c}{\sigma A^{2}} \cdot \frac{\alpha^{2}}{4}}
\approx 2.8^{\circ}[K]
$$

This temperature is in good agreement with the background
microwave-radiation from space. The de Broglie "thermostat" therefore
appears to be a very realistic notion.

It should be noted that the above approximate relation for radiation
temperature will give about the same result for a proton which radiates off
its energy via the electron layers in the hydrogen atom.

The relation between the absorbing and radiating surfaces has a basic form,
i.e. a function only of constants directly related to the vacuum space. The
background radiation could therefore also arise from sub-vacuum activities,
such as the spontaneous creation and absorption of electron-positron pairs
in accordance with the Dirac ether theory. The background radiation would
then be the visible sign of these processes, as well as the Universal
gravitational interaction.

If this is true, vacuum space itself could function as an absorbing medium
for electromagnetic energy. Whenever a photon intercepted a vacuum "hole"
it would lose one elementary quantum of energy while interacting with the
quantum volume that contains the "hole". One consequence of this is that
the cosmological redshift should be enhanced when electromagnetic waves
pass through areas of space with strong gravitational interactions, such as
in galaxies. This effect has been observed and has been analyzed by
Jaakkola (1977).
[end quote]

**Reply by HanDeBruijn** [reply]
Copied without permision and **with** rendering of the formulas (-:
>Good day Francois,
I find your speculations stimulating, and wonder if the following
exposition might resonate with your ideas:

>[begin quote]
We now return to de Broglie's thermodynamics, which, as noted earlier,
implies an exchange of energy between particles and the vacuum via a
"vacuum thermostat". This idea can be developed as follows.

>We saw earlier that the electron can be described by a model incorporating
a spherical surface for interactions with the surrounding space.

>This surface seems to be the locus of interactions at the quark level. It
is also the site where particle charge quanta are stored; it is the core of
rest-mass particles. The surface determines the inner structure of a
counterpart of a Schwarzschild singularity in the quantum space-time of
particles. We will therefore treat this area as an absolute sink for vacuum
energy. This is in accordance with the analysis of Carroll (1991) who
concludes that "in addition to the motion of particle spin, there exists a
source-sink flow at right angles to the motion of spin-through the vortex
centre".

>We assume that the vacuum space has an energy density $\rho_{0}$ (in terms
of mass).

>It was established above, in the section dealing with the force action,
that the vacuum tube corresponds to a surface which has a radius equal to
the Plank radius. We can therefore also calculate the corresponding
"graviton" mass density in the vacuum from the relation between surface and
mass:

$$
A=\frac{\text { Surface }}{\text { Mass }}=\frac{\Phi_{P l}}{\Phi_{P l}
\cdot c T_{0} \cdot \rho_{0}}
$$

>We thus have

$$
A \cdot c T_{0} \cdot \rho_{0}=1
$$

>The density is:

$$
\rho_{0}=\frac{1}{A \cdot c T_{0}} \equiv \frac{2 \pi G}{A^{2} c^{2}}
$$

>The energy inflow through the absorbing surface is:

$$
\frac{\mathrm{d} m_{e}}{\mathrm{~d} t}=c \rho_{0} \cdot A m_{e}
$$

>The increase of energy causes the quantum-system's temperature to increase,
and at an equilibrium temperature it will radiate off energy with a Planck
spectrum.

>As the radiating surface, we will use the surface corresponding to the time
constant in the electron system defined in the preceding section on the
fine structure constant. This time constant is assumed to be equal to the
length of the spiral in the electron system. The radiating surface is
consequently based on the quantum counterpart of the Compton wavelength:

$$
\Phi_{r a d}=\frac{1}{\pi} \ell^{2} \equiv
\frac{1}{\pi}\left(\frac{\hbar}{c m_{e}}\right)^{2}
$$

>The radiation temperature depends on the relation between the absorbing and
the radiating surfaces:

$$
\frac{\Phi_{a b s}}{\Phi_{r a d}}=\frac{A m_{e}}{\frac{1}{\pi} \ell^{2}}
\equiv \frac{\alpha^{2}}{4}
$$

>The temperature can be calculated with Stefan-Boltzmann's law:

$$
T=\sqrt[4]{\frac{\frac{\mathrm{d} m}{\mathrm{~d} t} \cdot c^{2}}{\sigma
\cdot \Phi_{\mathrm{rad}}}} \equiv \sqrt[4]{\frac{c^{3} \rho_{0}}{\sigma}
\cdot \frac{\Phi_{a b s}}{\Phi_{\mathrm{rad}}}}
$$

>The radiation temperature is, finally:

$$
T=\sqrt[4]{\frac{2 \pi G c}{\sigma A^{2}} \cdot \frac{\alpha^{2}}{4}}
\approx 2.8^{\circ}[K]
$$

>This temperature is in good agreement with the background
microwave-radiation from space. The de Broglie "thermostat" therefore
appears to be a very realistic notion.

>It should be noted that the above approximate relation for radiation
temperature will give about the same result for a proton which radiates off
its energy via the electron layers in the hydrogen atom.

>The relation between the absorbing and radiating surfaces has a basic form,
i.e. a function only of constants directly related to the vacuum space. The
background radiation could therefore also arise from sub-vacuum activities,
such as the spontaneous creation and absorption of electron-positron pairs
in accordance with the Dirac ether theory. The background radiation would
then be the visible sign of these processes, as well as the Universal
gravitational interaction.

>If this is true, vacuum space itself could function as an absorbing medium
for electromagnetic energy. Whenever a photon intercepted a vacuum "hole"
it would lose one elementary quantum of energy while interacting with the
quantum volume that contains the "hole". One consequence of this is that
the cosmological redshift should be enhanced when electromagnetic waves
pass through areas of space with strong gravitational interactions, such as
in galaxies. This effect has been observed and has been analyzed by
Jaakkola (1977).
[end quote]

**Comment by genotics** [comment]
Good day Han,
I own the copyright. THe excerpt is from the Broberg article of 1993. In
fact, I wrote parts of it.
Roy

**Comment by genotics** [comment]
Han, I neglected to thank you for the formulae. They appear correctly on
the journal website, where the file is built from scanned images.
R

**Comment by RedshiftDrift** [comment]
<100 years ago on this day, November 23, 1924, Edwin Hubble, then a thirty-five-year-old astronomer, had his findings first published in The New York Times, that there are distant stellar systems, that is, other “Island Universes”, beyond our own.>
https://mastodon.social/@pomarede/113533296264215013
